### DISCUSSION TITLE
TFDF serving in go

### DISCUSSION THREAD

**Discussion by shayansadeghieh** [google/yggdrasil-decision-forests#37] [Q&A] [answered]
Is the TFDF serving API in go still experimental? In the [README](https://github.com/google/yggdrasil-decision-forests/blob/main/yggdrasil_decision_forests/port/go/README.md) it claims to be experimental, but that was 6 months ago. 

Thank you!

**Comment by rstz** [comment, marked as answer]
Great question :)

We did not receive any bug reports on the Go implementations over the last 6 months and the API is now well-integrated in our continuous build system, so I believe that the implementation is stable on the engineering side.

The Go API still has some [limitations](https://github.com/google/yggdrasil-decision-forests/tree/main/yggdrasil_decision_forests/port/go#limitations); in particular, only a limited subset of models is supported by the Go API. 

More importantly, we did not get a lot of feedback on the API from the community - is it useful, what would you like to see next, what problems did people tackle with the API? Maybe we can use the discussion here to get a bit more insights and, if we see consensus that the API is indeed in a good state, I'm open so officially moving it out of its experimental state.
